src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -20,18 +20,11 @@ public abstract class AuthenticationManager
 
         public abstract IEnumerable<AuthenticationDescription> GetAuthenticationSchemes();
 
-        public abstract Task AuthenticateAsync(AuthenticateContext context);
+        public abstract Task<AuthenticateInfo> GetAuthenticateInfoAsync(string authenticationScheme);
```

Comment:
(oh, and `GetUserAsync` would be - almost - consistent with `HttpContext.User` and `ControllerBase.User` :smile:)